lib/y2_installbase.pm

```diff
@@ -0,0 +1,11 @@
+package y2_installbase;
+use base "installbasetest";
+use strict;
+use warnings;
+use y2_logs_helper;
+
+sub post_fail_hook {
+    y2_logs_helper->post_fail_hook;
```

Comment:
we should just move post_fail_hook from `y2_logs_helper`, and use imports for methods.

Reply:
do you want to move the function in the y2_installbase i guess. right?

Reply:
the post_fail_hook has been moved in the y2_installbase and the y2_logs_helper based have been replaced by that new module